examples/plot-graph-using-bokeh.ipynb

```diff
@@ -0,0 +1,200 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "65682475",
+   "metadata": {},
+   "source": [
+    "# Plot graph using `bokeh`\n",
```

Comment:
Are you going to add another notebook for plotly? I'd suggest to include it in the same notebook. You can also add it in the matplotlib example, that way users can view different plotting libraries in one place.

Reply:
Visualisation being a bigger theme, we want to create an example for each library. In the future, these notebooks can be grouped inside in a folder